contracts/ServiceRegistryL2.sol

```diff
@@ -29,7 +28,7 @@ contract ServiceRegistryL2 is GenericRegistry {
     event DrainerUpdated(address indexed drainer);
     event Deposit(address indexed sender, uint256 amount);
     event Refund(address indexed receiver, uint256 amount);
-    event CreateService(uint256 indexed serviceId);
+    event CreateService(uint256 indexed serviceId, bytes32 configHash);
```

Comment:
Corrected the event for service creation

Reply:
I am not sure this is needed

Reply:
We don't need to emit the `configHash` in other service related events other than the update event.

Reply:
Ok, as you both prefer then :-)